src/plotting/plot.component.tsx

```diff
@@ -121,7 +121,9 @@ const Plot = (props: PlotProps) => {
         },
         y2: {
           type: YAxesScale,
-          display: false,
+          display: selectedPlotChannels.some(
+            (channel) => channel.options.yAxis === 'right'
+          ),
```

Comment:
We should check this with CLF - there's a specific bit in the requirements to hide the right Y axis if nothing is assigned to it but nothing for the left one

Reply:
If we don't hide the left one, some values have to be assigned to it, which Chart.js does. And it potentially results in some confusion as it looks like one channel maps to two y-values. That was my reasoning

Reply:
True, should probably just ensure that this case never occurs

Reply:
Either that or give the user the freedom to plot on the right y-axis while hiding the left one (I know this isn't usually done in practice but in case they wanted to)